pyro/distributions/transforms/neural_autoregressive.py

```diff
@@ -7,13 +7,16 @@
 
 import torch
 import torch.nn as nn
+from pyro.distributions.conditional import ConditionalTransformModule
 from pyro.distributions.torch_transform import TransformModule
 from torch.distributions import constraints
 from torch.distributions.transforms import Transform, SigmoidTransform
 import torch.nn.functional as F
 
 from pyro.distributions.util import copy_docs_from
-from pyro.nn import AutoRegressiveNN
+from pyro.nn import AutoRegressiveNN, ConditionalAutoRegressiveNN
+
+from functools import partial
```

Comment:
nit: Can you use `isort` to automatically sort imports? That helps to keep our style canonical. You can probably find an editor plugin; otherwise run on the command line:
```py
pip install isort
isort -i  pyro/distributions/transforms/*.py
```

Reply:
Done!